packages/vue/src/components/molecules/SfColorPicker/SfColorPicker.vue

```diff
@@ -0,0 +1,61 @@
+<template>
+  <div class="sf-color-picker" :class="{ 'sf-color-picker--active': isOpen }">
+    <transition name="sf-color-picker" mode="out-in">
+      <div v-if="!isOpen" key="1" class="sf-color-picker__open">
+        <slot name="open">
+          <SfButton
+            class="color-secondary sf-color-picker__open--button"
```

Comment:
```suggestion
            class="color-secondary sf-color-picker__open-button"
```
but still we can find better name for this class 💚

Reply:
you're right! :) I'll change it